src/styles/custom.css

```
@@ -1,5 +1,5 @@
 /* Dark mode colors. */
-:root {
+:root[data-theme="dark"] {
```

```suggestion
:root {
```
Hvorfor ble den lagt til?